### ISSUE TITLE
Is it possible to define a Handle function which would catch all of the changes in a view-model?

### ISSUE THREAD

**Issue by miyconst** [Starcounter/Starcounter#3708]
Right now I can define a change handler in a view-model by a function:

```cs
void Handle(Input.MyProperty Action) {
}
```

Is there a way to catch changes in all properties? Something like:

```cs
void Handle(Input Action) {
    switch (Action.PropertyName) {
        case "Property0": break;
        case "Property1": /* Do something. */ break;
    }
}
```

**Comment by miyconst** [comment]
cc @chrhol

**Comment by chrhol** [comment]
No, there is no such way today. You can only add a handler for specific properties. Might be a good enhancement for the future though, but it's not currently in the roadmap.

**Comment by miyconst** [comment]
Is it difficult to implement? It would be really nice to have such feature. I have been asked about it a couple of time, including today by Modular Management guys.

**Comment by chrhol** [comment]
I guess the actual implementation would not be that difficult to implement, when we know how we want it. 

The main reason for waiting with it, is that there is a couple of quite serious issues concerning databinding, how metadata is set and allowing same instance for stateful partials in several places that keeps getting pushed back all the time. And these take some time to get in place. I would really like to have a plan and add new things in the end instead of having to switch and postpone all the time.

**Comment by miyconst** [comment]
@dan31 can we add this feature to the to do list?

**Comment by chrhol** [comment]
Part to review for #3727
